Source/WebKit/UIProcess/Inspector/win/WebInspectorUIProxyWin.cpp

```diff
@@ -77,6 +78,63 @@ static InspectedWindowInfo getInspectedWindowInfo(HWND inspectedWindow, HWND par
     return { rect.left, rect.top, rect.right - rect.left, rect.bottom - rect.top, parentRect.right - parentRect.left, parentRect.bottom - parentRect.top };
 }
 
+static String systemErrorMessage(DWORD errorCode)
+{
+    if (!errorCode)
+        return emptyString();
+
+    wchar_t* messageBuffer = nullptr;
+    size_t size = FormatMessageW(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
+        nullptr, errorCode, MAKELANGID(LANG_NEUTRAL, SUBLANG_NEUTRAL), (LPTSTR)&messageBuffer, 0, nullptr);
```

Comment:
Do you need `(LPTSTR)` cast here?
If so, use c++ style cast.

Reply:
It was unnecessary. Remove cast.

Reply:
Sorry, the cast was necessary. 
This function returns an `ERROR_INVALID_PARAMETER` error without a cast.
I'll use c++ style cast.

Reply:
Fixed at https://github.com/WebKit/WebKit/commit/70ce9358458f9401040946ef3392f2ae55d6fa3e
`reinterpret_cast<LPWSTR>(&messageBuffer)`